README.md

```diff
@@ -785,7 +785,9 @@ customButtons: [{
 }]
 ```
 
-- **customButtons.title** - String that is set as the button text
+- **customButtons.title** - String that is set as the button text (set only one of `title` OR `i18nKey`)
```

Comment:
update sample above for using i18nKey

Reply:
I've added a separate example without `title` but including `i18nKey` to avoid a confusing configuration where both are included.